internal/command/turboku/turboku.go

```diff
@@ -215,6 +215,7 @@ func run(ctx context.Context) error {
 	fmt.Fprintln(io.Out, "Dockerfile created")
 
 	appConfig.AppName = createdApp.Name
+	ctx = appconfig.WithName(ctx, appConfig.AppName)
```

Comment:
I'd add `ctx = appconfig.WithConfig(ctx, appConfig)`

and change line 186 from 
```
        appConfig := appConfig.NewConfig()
	appConfig.RawDefinition = createdApp.Config.Definition
```
to
```
     appConfig, err := appconfig.FromDefinition(&createdApp.Config.Definition)
```

This is what we are doing for `fly launch`